schema_to_cr.go

```diff
@@ -24,14 +24,14 @@ type CRD struct {
 	Group string
 }
 
-const cr_skeleton = `apiVersion: {{ .Group }}
+const cr_skeleton = `apiVersion: "{{ .Group }}/v1"
```

Comment:
It's really weird that the apiVersion isn't fully templated here. `.../v1` is the only possible api version this can create.